_blog/2023-12-15-using-actix-rust.mdx

```diff
@@ -1,23 +1,27 @@
 ---
-title: "Getting Started with Actix Web in Rust" 
-description: This article talks about how you can use actix-web to write a web application, covering routing, middleware, static files and databases.
+title: Getting Started with Actix Web in Rust
+description: This article talks about how you can use Actix Web to write a web application, covering routing, middleware, static files and databases.
 author: josh
-tags: [rust, actix-web, tutorial, guide]
-thumb: actix-rust-thumb.png 
+tags: [rust, actix-web, actix, tutorial, guide]
+thumb: actix-rust-thumb.png
 cover: actix-rust-thumb.png
 date: '2023-12-15T14:30:00'
 ---
 
-To this day, [`actix-web`](https://actix.rs/) remains an extremely formidable competitor in the Rust web backend ecosystem. It's no wonder that it's lasted so long; despite any impact that previous events may have had on it, it's still going strong and is one of the most recommended web frameworks in Rust. Originally based on the actor framework of the same name (`actix`), it has since moved away, and `actix` is now only really used for websocket endpoints. 
+To this day, [Actix Web](https://actix.rs/) remains an extremely formidable competitor in the Rust web backend ecosystem. It's no wonder that it's lasted so long; despite any impact that previous events may have had on it, it's still going strong and is one of the most recommended web frameworks in Rust. Originally based on the actor framework of the same name (`actix`), it has since moved away, and `actix` is now only really used for websocket endpoints.
 
-This article will primarily be concerning v4.4.0.
+This article will primarily be concerning v4.4.
 
 ## Getting Started with Actix Web
-To get started, you want to use `cargo init example-api` to generate your project, `cd` into the folder, and then use the following to install [`actix-web`](https://actix.rs/):
+
+To get started, you want to use `cargo init example-api` to generate your project, `cd` into the folder, and then use the following to add the `actix-web` crate to your project:
```

Comment:
```suggestion
To get started, you want to use `cargo init example-api` to generate your project, `cd` into the folder, and then use the following to add the [Actix Web](https://actix.rs/) crate to your project:
```

Reply:
This might be more consistent with the other changes in this pull request.

Reply:
It would except this is describing adding the crate to your Cargo.toml which requires this casing.